RevenueCatUI/Templates/V2/Localizations/LocaleExtensions.swift

```diff
@@ -37,6 +37,64 @@ extension Locale {
         #endif
     }
 
+    // swiftlint:disable:next identifier_name
+    var rc_languageScript: String? {
+        #if swift(>=5.9)
+        // `Locale.languageCode` is deprecated
+        if #available(macOS 13, iOS 16, tvOS 16, watchOS 9, visionOS 1.0, *) {
+            return self.language.script?.identifier
+        } else {
+            return self.scriptCode
+        }
+        #else
+        return self.scriptCode
+        #endif
+    }
+
+    // swiftlint:disable:next identifier_name
+    var rc_languageRegion: String? {
+        #if swift(>=5.9)
+        // `Locale.languageCode` is deprecated
+        if #available(macOS 13, iOS 16, tvOS 16, watchOS 9, visionOS 1.0, *) {
+            return self.language.region?.identifier
+        } else {
+            return self.regionCode
+        }
+        #else
+        return self.scriptCode
```

Comment:
I believe this should be
```suggestion
        return self.regionCode
```